osquery/tables/system/tests/darwin/certificates_tests.cpp

```diff
@@ -31,11 +31,20 @@ static std::string getCACertificateContent() {
   return content;
 }
 
+static std::string getCACertificateContentBigSerial() {
+  std::string content;
+  readFile(getTestConfigDirectory() / "test_cert_big_serial.pem", content);
+  return content;
+}
+
```

Comment:
I'm sure there's a more elegant way of doing a test for "certificates with serial #s > 8 bytes return the serial in hex" but we all lead busy lives.